retrofit/src/main/java/retrofit2/OkHttpCall.java

```diff
@@ -37,7 +38,7 @@
   @GuardedBy("this")
   private @Nullable okhttp3.Call rawCall;
   @GuardedBy("this")
-  private @Nullable Throwable creationFailure; // Either a RuntimeException, Error, or IOException.
```

Comment:
Isn't the comment still valid? Or the problem is that it doesn't really add any value?

Reply:
it could be any custom throwable, now/ (hopefully nobody's doing that, though.)